packages/build-info/package.json

```diff
@@ -16,9 +16,9 @@
   "scripts": {
     "prebuild": "rm -rf lib",
     "build": "tsc",
-    "test": "ava",
-    "test:dev": "ava -w",
-    "test:ci": "c8 -r lcovonly -r text -r json ava"
+    "test": "vitest run",
```

Comment:
What advantage does vitest have over Ava?

Reply:
Vitest has typescript support out of the box and was designed for typescript + ESM therefore its way faster and more relieable as well.

in the edge bundler we had similar issues that tests where timing out due to the node experimental `--loader` option that is needed for ava + typescript. Changing to vitests solved it: https://github.com/netlify/edge-bundler/pull/139

Reply:
My goal is to move all tests somewhere in the future to vitest